ositrace/ositrace_shacl.ttl

```diff
@@ -0,0 +1,286 @@
+            sh:node georeference:GeoreferenceShape ;
+            sh:order 7 ;
+            sh:path ositrace:georeference ] ;
+    sh:targetClass ositrace:OSITrace .
+
+ositrace:ContentShape a sh:NodeShape ;
+    sh:property [ skos:example "[motorway, residential]" ;
+            sh:datatype xsd:string ;
+            sh:description "Covered/used road types, defined over ODR element t_road_type, see ODR spec section 8.3"@en ;
+            sh:in ("Bicycle" "LowSpeed" "Motorway" "Pedestrian" "Rural" "Town" "TownArterial" "TownCollector" "TownExpressway" "TownLocal" "TownPlayStreet" "TownPrivate" "Unknown") ;
+            sh:message "Validation of roadTypes failed!"@en ;
+            sh:name "roadTypes"@en ;
+            sh:order 0 ;
+            sh:path ositrace:roadTypes ],
+        [ skos:example "shoulder, curb, ..." ;
+            sh:datatype xsd:string ;
+            sh:description "Covered lane types, see ODR spec section 9.5.3."@en ;
+            sh:in ("biking" "border" "connectingRamp" "curb" "driving" "entry" "exit" "median" "none" "offRamp" "onRamp" "parking" "restricted" "shoulder" "slipLane" "stop" "walking") ;
+            sh:message "Validation of laneTypes failed!"@en ;
+            sh:name "laneTypes"@en ;
+            sh:order 1 ;
+            sh:path ositrace:laneTypes ],
+        [ skos:example "trees, street lamps, road patches..." ;
+            sh:datatype xsd:string ;
+            sh:description "Covered object classes, see ODR spec section 11"@en ;
+            sh:in ("barrier" "bike" "building" "bus" "car" "crosswalk" "gantry" "motorbike") ;
+            sh:message "Validation of levelOfDetail failed!"@en ;
+            sh:name "levelOfDetail"@en ;
+            sh:order 2 ;
+            sh:path ositrace:levelOfDetail ], 
+        [ skos:example "right-hand traffic" ;
+            sh:datatype xsd:string ;
+            sh:description "Traffic direction, i.e. right-hand or left-hand traffic"@en ;
+            sh:in ("left-hand" "right-hand") ;
+            sh:maxCount 1 ;
+            sh:message "Validation of trafficDirection failed!"@en ;
+            sh:name "trafficDirection"@en ;
+            sh:order 3 ;
+            sh:path ositrace:trafficDirection ],
+        [ skos:example "object list, detection list" ;
+            sh:datatype xsd:string ;
+            sh:description "Level of granularity of sensor data"@en ;
+            sh:in ("object list" "detection list") ;
+            sh:message "Validation of granularity failed!"@en ;
+            sh:minCount 1 ;
+            sh:name "granularity"@en ;
+            sh:order 4 ;
+            sh:path ositrace:granularity ],
+        [ skos:example "119" ;
+            sh:datatype xsd:string ;
+            sh:description "Identifier of scenario performed in the trace file"@en ;
+            sh:message "Validation of scenarioIdentifier failed!"@en ;
+            sh:name "scenarioIdentifier"@en ;
+            sh:order 5 ;
+            sh:path ositrace:scenarioIdentifier ],
```

Comment:
When we want to express an direct relationship to an `Scenario` (https://github.com/GAIA-X4PLC-AAD/ontology-management-base/blob/main/scenario/scenario_ontology.ttl), we should use `xsd:anyURI` and the `did` of the scenario. However, I think this a loose interaction or more internal identifier, right? Maybe rename the `path` to `ositrace:traceIdentifier`.